File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.cs
public class Solution {
    public int CountEven(int num) {
        int count=0;
        for(int i=2;i<=num;i++){
            int sum=0;
            int val=i;
            while(val>0){
                sum+=val%10;
                val=val/10;
            }
            if(sum%2==0){
                count++;
            }
        }
        return count;
        
    }
}

File: 2298-count-integers-with-even-digit-sum/2298-count-integers-with-even-digit-sum.js
/**
 * @param {number} num
 * @return {number}
 */
var countEven = function(num) {
   let count=0
   for(let i=1;i<=num;i++){
    let str=i.toString()
    let sum=0
    for(let j=0;j<str.length;j++){
        sum+=Number(str[j])
    }
    if(sum%2===0){
        count++
    }
   }
   return count
};